File: src/main/java/uk/edu/populationfitness/models/genes/fitness/Statistics.java
package uk.edu.populationfitness.models.genes.fitness;

public class Statistics {
    private double min = Double.MAX_VALUE;

    private double max = Double.MIN_VALUE;

    public void add(double value){
        min = Math.min(min, value);
        max = Math.max(max, value);
    }

    public void show(){
        System.out.println("Min="+min+" Max="+max);
    }
}

File: DotNet/PopulationFitness/PopulationFitness/Models/Genes/Fitness/Statistics.cs
using System;

namespace PopulationFitness.Models.Genes.Fitness
{
    public class Statistics
    {
        private double _min = double.MaxValue;

        private double _max = double.MinValue;

        public void Add(double value)
        {
            _min = Math.Min(_min, value);
            _max = Math.Max(_max, value);
        }

        public void Show()
        {
            Console.WriteLine("Min=" + _min + " Max=" + _max);
        }
    }
}
